logger/logger.go

```diff
@@ -0,0 +1,38 @@
+package logger
+
+import (
+	log "github.com/sirupsen/logrus"
+	"os"
+)
+
+func init() {
+	log.SetFormatter(&log.JSONFormatter{})
+	log.SetOutput(os.Stdout)
+	log.SetLevel(log.InfoLevel)
```

Comment:
should probably set it based on an env variable, so that we can turn this to `info` on demand

Reply:
sure will take it up as part of the refactoring